lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -571,10 +600,10 @@ def compute_initial_guess(
 
         if (predictor_of_mean_flag.lower() == "mean" and
                 not estimate_coefficients_from_linear_model_flag):
-            initial_guess = [1, 1, 0, 1]
+            initial_guess = [0, 1, 0, 1]
```

Comment:
Good.  Have you tested how far sensible initialisation goes towards achieving stable results?

Reply:
This question has been addressed elsewhere.